target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -35,6 +35,11 @@ engenius,eap1300)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "${boardname}:yellow:wlan5g" "phy1tpt"
 	ucidef_set_led_default "mesh" "MESH" "${boardname}:blue:mesh" "0"
 	;;
+engenius,eap2200)
+	ucidef_set_led_netdev "lan1" "LAN1" "${boardname}:green:lan1" "eth0"
+	ucidef_set_led_netdev "lan2" "LAN2" "${boardname}:green:lan2" "eth1"
+	ucidef_set_led_default "mode" "MODE" "${boardname}:blue:mode" "0"
```

Comment:
Isn't that just "default-state: off" for the LED in DTS?
Setting it to off is default anyway for other targets, so maybe one can just remove this line?

Reply:
I removed it.
Thanks for advice.